readme.md

```diff
@@ -209,6 +209,10 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 
 ## Changelog
 
+### 2.1.3 (19 February 2021)
+- Fix issue JENKINS-63009 (Jobs now work with folder credentials)
+- Fix issues JENKINS-61182 and JENKINS-62789 
```

Comment:
[JENKINS-62789](https://issues.jenkins.io/browse/JENKINS-62789) is "htmlpublisher plugin may break with tables-to-divs". Perhaps you meant [JENKINS-62798](https://issues.jenkins.io/browse/JENKINS-62798) "WEBhook return error 400 when trigered from push".